### ISSUE TITLE
Can you update the case?

### ISSUE THREAD

**Issue by jziancheng** [komorra/NodeEditorWinforms#16]
Can you update or  add  more cases?

**Comment by komorra** [comment]
What do you mean by cases?

**Comment by jziancheng** [comment]
it is sample. like math sample



---Original---
From: ***@***.***&gt;
Date: Wed, May 19, 2021 01:32 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [komorra/NodeEditorWinforms] Can you update the case? (#16)




 
What do you mean by cases?

**Comment by komorra** [comment]
Yes, in the future. But I don't know when exactly. What samples are you interested the most? (I considered audio or bitmap samples to implement)

**Comment by jziancheng** [comment]
> Yes, in the future. But I don't know when exactly. What samples are you interested the most? (I considered audio or bitmap samples to implement)

would you update the example like the picture in the github?

**Comment by jziancheng** [comment]
> Yes, in the future. But I don't know when exactly. What samples are you interested the most? (I considered audio or bitmap samples to implement)

The math example is too simple to show all the functions of NodeEditor
